lib/tasks/asset_manager.rake

```diff
@@ -1,6 +1,9 @@
 namespace :asset_manager do
-  desc "Migrates Organisation logos to Asset Manager."
-  task migrate_organisation_logos: :environment do
-    MigrateAssetsToAssetManager.new.perform
+  desc "Migrates Assets to Asset Manager."
+  task :migrate_assets, [:target_dir] => :environment do |_, args|
+    raise "Usage: asset_manager:migrate_assets[system/uploads/organisation/logo]" unless args[:target_dir]
```

Comment:
I think using "/system/uploads/organisation/logo" as an example is a bit confusing. Can you think of a way of being clearer about what this argument should be? Or do you think it's OK as it is?

Reply:
Thanks @chrisroos - I've improved the documentation string and will rebase before merging.